src/Components/Web.JS/src/Boot.WebAssembly.ts

```diff
@@ -5,14 +5,14 @@
 import { DotNet } from '@microsoft/dotnet-js-interop';
 import { Blazor } from './GlobalExports';
 import * as Environment from './Environment';
-import { byteArrayBeingTransferred, Module, BINDING, monoPlatform } from './Platform/Mono/MonoPlatform';
+import { Module, BINDING, monoPlatform } from './Platform/Mono/MonoPlatform';
```

Comment:
In next PR we could try re-invent how the goal of `invokeJSFromDotNet` could be achieved otherwise. 
To ged rid of things like `BINDING.js_string_to_mono_string`